docs-src/concepts/what-is-advanced-visibility.md

```diff
@@ -9,9 +9,10 @@ tags:
   - visibility
 ---
 
-Advanced Visibility, within the Temporal Platform, is the subsystem and APIs that enable the listing, filtering, and sorting of Workflow Executions through a custom SQL-like [List Filter](/concepts/what-is-a-list-filter).
+Advanced Visibility, within the Temporal Platform, is the subsystem and APIs that enable the listing, filtering, and sorting of [Workflow Executions](/concepts/what-is-a-workflow-execution) through a custom SQL-like [List Filter](/concepts/what-is-a-list-filter).
 
-- In Temporal Cluster version 1.20 and later, advanced Visibility is available on SQL databases like MySQL (version 8.0.17 and later) and PostgreSQL (version 12 and later), in addition to support for Elasticsearch.
-- For Temporal Server versions 1.19.1 and earlier, you must [integrate with ElasticSearch](/clusters/how-to-integrate-elasticsearch-into-a-temporal-cluster) to use advanced Visibility.
+- In Temporal Cluster version 1.20 and later, Advanced Visibility is available on SQL databases like MySQL (version 8.0.17 and later) and PostgreSQL (version 12 and later), in addition to support for Elasticsearch.
+- For Temporal Server versions 1.19.1 and earlier, you must [integrate with ElasticSearch](/clusters/how-to-integrate-elasticsearch-into-a-temporal-cluster) to use Advanced Visibility.
   Elasticsearch takes on the Visibility request load, relieving potential performance issues.
   We highly recommend operating a Temporal Cluster with Elasticsearch for any use case that spawns more than just a few Workflow Executions.
+- On Temporal Cloud, [there is a read-only role assignable at account and Namespace level](/cloud-context/get-started-certificates) to enable Advanced Visibility.
```

Comment:
Also I think advanced Visibility is enabled by default for all Cloud users because there is no standard Visibility as such in Cloud. Would you please check?